es-vue-base/src/lib-components/EsNavBarTopLevelMenu.vue

```diff
@@ -5,43 +5,33 @@
         <div class="top-header-inner w-100">
             <!-- desktop fly-out menu trigger -->
```

Comment:
We should update this comment to be descriptive of what this element ends up rendering, as it's no longer only fly-out menus anymore.